rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -1696,6 +1696,9 @@ object RudderConfig extends Loggable {
           case BooleanVType => new CheckboxField(id)
           case TextareaVType(r) => new TextareaField(id)
           case PasswordVType(algos) => new PasswordField(id, input.constraint.mayBeEmpty, algos)
+          case MasterPasswordVType(algos) => new MasterPasswordField(id, input.constraint.mayBeEmpty, algos)
+          case AixSlavePasswordVType => new SlavePasswordField(id, HashAlgoConstraint.SlavePasswordType.AIX)
```

Comment:
these are not the same type as in https://github.com/Normation/cf-clerk/pull/84/files#diff-0a14b9750ca44a8ca19cb95baae3b5d0R84